<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secret Santa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#00743c;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>641</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:#AE0D13;
border: 3px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>601</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>449</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="titolo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TITOLO</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:#AE0D13;
	border: 3px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="form_nome">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:white;
border: 3px solid black;
border-radius: 10px;
color:black
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> Nome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:white;
	color: black;
	border:3px solid black;
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>AGGIUNGI</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="form_email">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:white;
border: 3px solid black;
border-radius: 10px;
color:black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> Email</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="errorMsg">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit{
	background-color:#AE0D13;
	color:#AE0D13;
	font-size:15px;
	font-weight:bold;
	border:none;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:15px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:400;&quot;&gt;Compila tutti i campi del form&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="InviaEmail">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:white;
	color: black;
	border:3px solid black;
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>INVIA EMAIL</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="bells">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="christmas_hat">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
